tests/unittest/inputs/test_async_media_loading.py

```
@@ -0,0 +1,268 @@
+- async_load_image / async_load_audio with data URLs and file paths
+- CPU-bound work is offloaded to a thread pool (event loop not blocked)
+- aiohttp session is reused across calls (_get_aiohttp_session)
+- MultimodalDataTracker.retrieve_all_async gathers all modalities concurrently
+"""
+
+import asyncio
+import base64
+import tempfile
+import threading
+import time
+from collections import defaultdict
```

Flake8 (7.3.0)

[error] 'collections.defaultdict' imported but unused

(F401)